src/sentry/utils/committers.py

```diff
@@ -254,7 +246,11 @@ def get_serialized_event_file_committers(project, event, frame_limit=25):
 
     for committer in committers:
         commit_ids = [commit.id for (commit, _) in committer["commits"]]
-        committer["commits"] = [serialized_commits_by_id[commit_id] for commit_id in commit_ids]
+        commits_result = [serialized_commits_by_id[commit_id] for commit_id in commit_ids]
+        # Deduplicate commits
+        committer["commits"] = [
+            i for n, i in enumerate(commits_result) if i not in commits_result[n + 1 :]
```

Comment:
@NisanthanNanthakumar Isn't the runtime of this going to be O(n^2)? How large is n here? I wonder if there is a way to do this more efficiently. Perhaps by building a set of already seen commits?.